microsoft/testsuites/dpdk/dpdkvpp.py

```diff
@@ -84,6 +84,7 @@ def run_test(self) -> None:
         vpp_detected_interface = (
             "GigabitEthernet" in vpp_interface_output
             or "VirtualFunctionEthernet" in vpp_interface_output
+            or "NetVSC" in vpp_interface_output
```

Comment:
NetVSC is expected or not, can you double confirm? from the text, I understand it is synthetic network?